src/doc/src/reference/unstable.md

```diff
@@ -12,9 +12,9 @@ config file (`.cargo/config.toml`) in the `unstable` table. For example:
 
 ```toml
 [unstable]
-mtime-on-use = 'yes'
-multitarget = 'yes'
-timings = 'yes'
+mtime-on-use = true
+multitarget = true
+timings = true
```

Comment:
I don't think this value can take a boolean, maybe something like this?

```suggestion
timings = ["html"]
```

Reply:
Yeah, that's correct.